src/app/zap-templates/zcl/data-model/chip/closure-dimension-cluster.xml

```diff
@@ -0,0 +1,261 @@
+  </enum>
+
+  <enum name="RotationAxisEnum" type="enum8">
+    <cluster code="0x0105"/>
+    <item name="Left" value="0x00"/>
+    <item name="CenteredVertical" value="0x01"/>
+    <item name="LeftAndRight" value="0x02"/>
+    <item name="Right" value="0x03"/>
+    <item name="Top" value="0x04"/>
+    <item name="CenteredHorizontal" value="0x05"/>
+    <item name="TopAndBottom" value="0x06"/>
+    <item name="Bottom" value="0x07"/>
+    <item name="LeftBarrier" value="0x08"/>
+    <item name="LeftAndRightBarriers" value="0x09"/>
+    <item name="RightBarrier" value="0x0A"/>
+  </enum>
+
+  <enum name="StepDirectionEnum" type="enum8">
+    <cluster code="0x0105"/>
+    <item name="Decrease" value="0x00"/>
+    <item name="Increase" value="0x01"/>
+  </enum>
+
+  <enum name="TargetLatchEnum" type="enum8">
+    <cluster code="0x0105"/>
+    <item name="Latch" value="0x00"/>
+    <item name="Unlatch" value="0x01"/>
+  </enum>
+
+  <enum name="TranslationDirectionEnum" type="enum8">
+    <cluster code="0x0105"/>
+    <item name="Downward" value="0x00"/>
+    <item name="Upward" value="0x01"/>
+    <item name="VerticalMask" value="0x02"/>
+    <item name="VerticalSymmetry" value="0x03"/>
+    <item name="Leftward" value="0x04"/>
+    <item name="Rightward" value="0x05"/>
+    <item name="HorizontalMask" value="0x06"/>
+    <item name="HorizontalSymmetry" value="0x07"/>
+    <item name="Forward" value="0x08"/>
+    <item name="Backward" value="0x09"/>
+    <item name="DepthMask" value="0x0A"/>
+    <item name="DepthSymmetry" value="0x0B"/>
+    <item name="CeilingSimple" value="0x0C"/>
+    <item name="CeilingMask" value="0x0D"/>
+    <item name="CeilingSidedSymmetry" value="0x0E"/>
+    <item name="CeilingCenteredSymmetry" value="0x0F"/>
+  </enum>
+
+  <struct name="CurrentStruct" apiMaturity="provisional">
+    <cluster code="0x0105"/>
+    <item fieldId="0" name="Position" type="percent100ths" optional="true"/>
+    <item fieldId="1" name="Latching" type="LatchingEnum" optional="true" default="0x02" min="0x00" max="0x02"/>
+    <item fieldId="2" name="Speed" type="ThreeLevelAutoEnum" optional="true" default="0x00" min="0x00" max="0x03"/>
+  </struct>
+
+  <struct name="RangePercent100thsStruct" apiMaturity="provisional">
+    <cluster code="0x0105"/>
+    <item fieldId="0" name="Min" type="percent100ths"/>
+    <item fieldId="1" name="Max" type="percent100ths" max="10000"/>
+  </struct>
+
+  <struct name="TargetStruct" apiMaturity="provisional">
+    <cluster code="0x0105"/>
+    <item fieldId="0" name="Position" type="percent100ths" optional="true"/>
+    <item fieldId="1" name="Latch" type="TargetLatchEnum" optional="true" default="0x01" min="0x00" max="0x01"/>
+    <item fieldId="2" name="Speed" type="ThreeLevelAutoEnum" optional="true" default="0x00" min="0x00" max="0x03"/>
+  </struct>
+
+  <struct name="UnitRangeStruct" apiMaturity="provisional">
+    <cluster code="0x0105"/>
+    <item fieldId="0" name="Min" type="int16s"/>
```

Comment:
I know its define like this in the spec. But shouldn't the min have the same upper constraint as the max? `32767`

Reply:
The changes should come from the specification, so we will raise this as a spec issue and track it.

Reply:
They do have the same upper constraint, they are just maximally non-obvious about it.

The constraint for Max should have been written as `min Min`

Reply:
Will discuss with Ali and fix it in spec